### ISSUE TITLE
2024_feb_workflow #story_b: DEV Causal Loop Diagrams

### ISSUE THREAD

**Issue by lijenn** [lzim/teampsd#3085]
Copied from #3024 

--------
## Team Care Flow Diagram Wireframe

![image](https://github.com/lzim/teampsd/assets/30132017/c7bbf347-3fe7-4aad-8d05-b54e9a49bed6)

![image](https://github.com/lzim/teampsd/assets/30132017/cb5810f1-cce3-4492-92a9-bb5aeab54c15)

![image](https://github.com/lzim/teampsd/assets/30132017/b9c117f5-0935-4a95-a054-b1313f2d083b)

![image](https://github.com/lzim/teampsd/assets/30132017/b49e3b1a-14a1-4aea-8813-664e10470a8f)

### Acceptance Criteria

- [ ] When the user clicks on the "Flow" icon from the Dynamic Decision Dashboard, the user is presented with a new browser tab with the diagram depicted in the wireframe.
- [ ] The flows and stock variables are proportional to each other based on the instructions in the wireframe.
- [ ] Flows are labeled with actual values. 
- [ ] Stocks change color based on rules depicted in the wireframe.

**Comment by jamesmrollins** [comment]
@lzim 

- [ ] We will begin development of these 7/8.
- [ ] We will have MVP in test by 7/17.
- [ ] @lijenn will test 7/18-19.
- [ ] Planned MVP into PROD by 7/22.

**Note:** The MVP means that the flow diagrams work as designed but may not support scaling or dynamic window sizing.  This will take considerable effort and may require several more weeks to accomplish. I also need to verify that scaling is possible (it should be because are using SVG).

**Comment by jamesmrollins** [comment]
@lzim @ljmoody @lijenn 

## Sim UI Workflow 7/3

- [ ] CC, MM, and Psy are available on Test Slow.
- [ ] They do not have dynamic sizing available in this version.
- [ ] Will report as more become available.

**Comment by jamesmrollins** [comment]
@tejaspesquare Please provide a weekending update for this effort.  Thank you.

**Comment by tejaspesquare** [comment]
Hi @jamesmrollins,

Development for Team Care Flow meter is currently in progress. The flow diagram has been completed, and work is underway to build the logic for rendering data by developer. Hoping, it should be ready by Monday EOD.

just FYI - This development is a collaborative effort between the Designer and Developer. Designer is also engaged in other internal assignments hence not able to join all the time for this project.

![image](https://github.com/user-attachments/assets/a9798a50-b856-41de-9c81-f538f3ae5578)
![image](https://github.com/user-attachments/assets/18a44f0f-4d76-4a43-a269-2554a3fe7129)

Note: Flow Diagram for Team Flow designing part is under way from Designer.

**Comment by matthewtomo** [comment]
**Discussed DEV Standup 7/29/2024**
@matthewtomo @tejaspesquare 

- Dev team now has the corrected list of variable names for the Team Care Model and will be working on verifying that they can find/access them in the API calls to Forio Epicenter.
- Once Dev team verifies the list of variables for Team Care, they will move onto the Team Flow Model.
- Design work is currently in progress for Team Flow Diagram.

fyi: @jamesmrollins @lijenn @ljmoody

**Comment by matthewtomo** [comment]
Discussed in DEV Standup 8/1/2024
@matthewtomo @tejaspesquare

- The corrected variables I sent back to Tejas were not found in the API
  - DEV can not directly access the VMFX file 
  - API Results do not capture these variables
  - This issue is affecting the AGG model

fyi: @jamesmrollins @lijenn @ljmoody

**Comment by tejaspesquare** [comment]
Hello @ljmoody , @lijenn , @matomasz 

> * Design work is currently in progress for Team Flow Diagram.

The design phase for Team Flow has been finished. Developer will proceed to next stage of development once Team Care is completed. 

FYI: @jamesmrollins

**Comment by matthewtomo** [comment]
**Discussed SDM-DOC CI/CD meeting 8/6/2024** 
@matthewtomo @jamesmrollins 

- @tejaspesquare can you confirm which variable names are not being captured in your API calls?
- When @jamesmrollins and I validated the model two weeks ago, all of the variable names were accounted for.

**Comment by tejaspesquare** [comment]
Hi Matthew,

> * @tejaspesquare can you confirm which variable names are not being captured in your API calls?

[Missing Variables- Flow Meter Development for Team Care (Agg)_Updated 7thAug.txt](https://github.com/user-attachments/files/16519808/Missing.Variables-.Flow.Meter.Development.for.Team.Care.Agg._Updated.7thAug.txt)

Attached updated list of variables associated with Agg (Team Care) model. Some of variables present in API response which were not earlier.

Example: For CC -> CC Evaluation Rate is not found in API call
![image](https://github.com/user-attachments/assets/eb26a9d2-cc44-45c3-bd52-17301fc59b2a)

**Comment by tejaspesquare** [comment]
Hi @matomasz,

Attached is a list of missing variables that are associated with Team flow. 
[MIssing Variables-Flow Meter Development for Team Flow(SP).txt](https://github.com/user-attachments/files/16521443/MIssing.Variables-Flow.Meter.Development.for.Team.Flow.SP.txt)

**Comment by matthewtomo** [comment]
**Discussed DEV standup 8/9/2024**
@matthewtomo @tejaspesquare 

- List of corrected variables for Agg is here: 
[Correct Variables- Flow Meter Development for Team Care (Agg) (1).txt](https://github.com/user-attachments/files/16561702/Correct.Variables-.Flow.Meter.Development.for.Team.Care.Agg.1.txt)
- @matthewtomo will double check that the Sim UI DEV environment has the most up to date model files, and then notify here once completed.
- Then, @tejaspesquare will check for the variables again.

**Comment by ljmoody** [comment]
@matthewtomo, placing these suggested graphical improvements for flow here (originally from a comment in #3168). This-and other suggestions-may require continued design development and discussions between sim_ui & Lindsey.

---
### Graphical improvements
- [ ] 1. The flows "bend" to/from the flows, creating unnecessary visual complexity AND challenges when zooming in to read the small text values. Recommend that there are no turns in the flows, just parallel left to right viewing to/from the stocks.
- [ ] 2. The stocks require a scale indicator on the tick marks.

![image](https://github.com/user-attachments/assets/c45d472a-fd94-4037-b030-32d723fa15e6)

**Comment by matthewtomo** [comment]
> **Discussed DEV standup 8/9/2024** @matthewtomo @tejaspesquare
> 
> * List of corrected variables for Agg is here:
>   [Correct Variables- Flow Meter Development for Team Care (Agg) (1).txt](https://github.com/user-attachments/files/16561702/Correct.Variables-.Flow.Meter.Development.for.Team.Care.Agg.1.txt)
> * @matthewtomo will double check that the Sim UI DEV environment has the most up to date model files, and then notify here once completed.
> * Then, @tejaspesquare will check for the variables again.

@tejaspesquare the model files are now up to date for the sim UI in Forio Epicenter DEV

**Comment by matthewtomo** [comment]
**Discussed DEV Standup 8/12/24**
@matthewtomo @tejaspesquare 

- There is currently an error occurring when attempting to access the AGG model file through the API (see error message below)
- This error is not occurring with any of the other model files
- @matthewtomo will provide @tejaspesquare a list of all changed variable names that were included in the last round of model file updates

fyi: @jamesmrollins 

Error Message:
```
{
  "trace": [
    {
      "type": "python",
      "function": "load_model",
      "file": "/usr/local/lib/python3.10/dist-packages/epicenter/worker/vensim/vensim_worker.py",
      "line": 58
    },
    {
      "type": "python",
      "function": "load_model",
      "file": "/usr/local/lib/python3.10/dist-packages/epicenter/worker/vensim/vensim_wrapper.py",
      "line": 32
    },
    {
      "type": "python",
      "function": "vensim_command",
      "file": "/usr/local/lib/python3.10/dist-packages/epicenter/worker/vensim/vensim_wrapper.py",
      "line": 176
    }
  ],
  "information": {
    "runKey": "0000019146a0e46b62a2c400189e0065fd20",
    "code": "MODEL_INITIATION"
  },
  "type": "python",
  "message": "VensimLibraryException: Vensim command(SPECIAL\u003ELOADMODEL|/home/epicenter/model/0000019146a0e46b62a2c400189e0065fd20/agg_PROD.vmfx) failed)"
```

**Comment by tejaspesquare** [comment]
Hi @matomasz

> Attached is a list of missing variables that are associated with Team flow. [MIssing Variables-Flow Meter Development for Team Flow(SP).txt](https://github.com/user-attachments/files/16521443/MIssing.Variables-Flow.Meter.Development.for.Team.Flow.SP.txt) 

> Note: We are not sure how this match will be working (SIM UI) GMH Recommend Step up Rate -> (API Response) PC/PCMHI to GMH Recommend Step up Rate

Do we have any updated on this?

**Comment by matthewtomo** [comment]
**Discussed Dev Standup8/14/2024**
@matthewtomo @tejaspesquare 

- @matthewtomo will investigate with @jamesmrollins to see what variables should included for the Team Flows (SP) and check the match: (SIM UI) GMH Recommend Step up Rate -> (API Response) PC/PCMHI to GMH Recommend Step up Rate

**Comment by matthewtomo** [comment]
**Discussed 8/15/2024**
@matthewtomo @jamesmrollins 

- Flows should be based on each of the different models, not services for Team Care (illustrated below for MM):
![image](https://github.com/user-attachments/assets/c13c19d4-4d3e-4d38-aea4-58c92e212915)
- MM Flow variables should be taken from the MM model **(NOT Agg Model!)**

**Comment by matthewtomo** [comment]
**Discussed #DEV_Standup**
@matthewtomo @tejaspesquare 

- Discussed the required variables for Team Care Flow Diagram:
![image](https://github.com/user-attachments/assets/0ff6f8d6-678f-445b-a1af-599e942e7e01)
- The **GROUND TRUTH for information** regarding this feature is the wireframes available here (Team Care is page 10): https://takouba.sharepoint.com/sites/TeamPraxis/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FTeamPraxis%2FShared%20Documents%2FGeneral%2FWireframes%2FSim%20UI%20Release%203%2E7%20all%2Epdf&viewid=0ddbc094%2D7c2d%2D4d1e%2Da7e3%2D4805a6de605d&q=Agg&parent=%2Fsites%2FTeamPraxis%2FShared%20Documents%2FGeneral&parentview=7
- Discussed how _Intake Evaluation Rate_,  _Patients Waiting for Intake Evaluation_, and _Intake Completion Rate_ are not split up into sub rates (MM, CC, etc.) unlike other rates in the flow.
- _Intake_ is **NOT a sub branch for the following variables** in the flow diagram: _Patients Waiting to Start a Service_, _Patients in Service_, _Treatment Decision Rate_, _Starting a Service Rate_, _Ending Service Rate_
- @matthewtomo will verify with  James that we will use the Modified Agg file.
- @tejaspesquare will updated the developer working on this feature.

**Comment by matthewtomo** [comment]
**wk3 2024_08 Dev Standup**
@matthewtomo @tejaspesquare

- Discussed that this feature should be top priority as of right now.

**Comment by matthewtomo** [comment]
**Discussed Dev Standup wk4 2024_08**
@matthewtomo @tejaspesquare 

- For this week, the top priority for DEV should be finishing the Flow diagrams, for all of the models.
- @matthewtomo will investigate the new Agg issues and report back here: https://github.com/lzim/teampsd/issues/3199#issuecomment-2312440559

**Comment by tejaspesquare** [comment]
Hello @jamesmrollins , @matomasz 

Team Care (Agg) development is completed & can be verified on Dev instance. 
Fig #1
![image](https://github.com/user-attachments/assets/13c9a310-0d8d-4b83-9e31-7da1ef667d28)

Fig #2
![image](https://github.com/user-attachments/assets/4eaa9e79-0c3f-4420-b48b-81aee174dea9)

Note: Fig #2 - We have noticed under patients -> Patients in Service appointments figure is overlapped with blue bar box.


FYI - @ljmoody , @lijenn

**Comment by matthewtomo** [comment]
@tejaspesquare
- It does not look like the bars connecting stocks and flows are properly proportioned.
- The large blue areas are bars with widths of over 20000 px in the css styling tab.
- According to the wireframes, all of the parallel flows should take up a max of 75 px total (https://takouba.sharepoint.com/sites/TeamPraxis/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FTeamPraxis%2FShared%20Documents%2FGeneral%2FWireframes%2FSim%20UI%20Release%203%2E7%20all%2Epdf&viewid=0ddbc094%2D7c2d%2D4d1e%2Da7e3%2D4805a6de605d&q=Agg&parent=%2Fsites%2FTeamPraxis%2FShared%20Documents%2FGeneral&parentview=7).
![image](https://github.com/user-attachments/assets/9c06bb11-5b4d-4111-8b94-d4b5285fb92d)
![image](https://github.com/user-attachments/assets/935238ac-9981-4223-a7cc-d7dd2252f290)

**Comment by tejaspesquare** [comment]
Hi Matthew
> * It does not look like the bars connecting stocks and flows are properly proportioned.
> * The large blue areas are bars with widths of over 20000 px in the css styling tab.

The issue that was identified yesterday has been resolved. Could you kindly verify again using multiple model files for MM & Team Care (Agg), if feasible?

1. Team Care(Agg) 
- The Booking Rate and Completing Rate reflect the actual flow value, with the Dialmax value maintained at 100 in a code; otherwise, it would have been set to 50 for both dials.
![image](https://github.com/user-attachments/assets/5f581c37-fb05-4985-b942-16c56a348865)

- The value is not displayed at all. There is still work to be done to resolve this issue.
![image](https://github.com/user-attachments/assets/760474ac-7b5b-43e9-805a-8349752b83a1)


2. MM
Appointments
![image](https://github.com/user-attachments/assets/233138b1-8b3a-4f03-b42c-b163b1549ecb)

    Patients
   ![image](https://github.com/user-attachments/assets/96e0221d-cabb-4fb3-89d1-065c5b7ad2c7)


Thank You!

**Comment by matthewtomo** [comment]
@tejaspesquare, here is my update for the SP variables. I have a few I need to check with James about and a few that I was able to provide you names for: 
[Corrected.Variables-Flow.Meter.Development.for.Team.Flow.SP.txt](https://github.com/user-attachments/files/16809325/Corrected.Variables-Flow.Meter.Development.for.Team.Flow.SP.txt)

Everything after this is notes/explanation for what I did to track these down. No need to read all of it, but it might help make better sense of the issues I need to address with James.

It was a little confusing to figure out which variables you were referring to. The three categories exist in a ranked order:
- SMH
- GMH
- PC/PCMHI

For example, you can step up from PC/PCMHI to GMH, from PC/PCMHI to SMH, or from GMH to SMH. These are all steps up because they go up this ranking. There is no step up from SMH as there are no higher rankings. Following the logic in the opposite direction, you can step down from SMH to GMH, from SMH to PC/PCMHI, or from GMH to PC/PCMHI. There is no step down from PC/PCMHI as there are no lower rankings.

I will copy over a few photos from the [wireframes (page 9)](https://takouba.sharepoint.com/sites/TeamPraxis/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FTeamPraxis%2FShared%20Documents%2FGeneral%2FWireframes%2FSim%20UI%20Release%203%2E7%20all%2Epdf&viewid=0ddbc094%2D7c2d%2D4d1e%2Da7e3%2D4805a6de605d&q=Agg&parent=%2Fsites%2FTeamPraxis%2FShared%20Documents%2FGeneral&parentview=7) and discuss what I was able to find.

## **Team Flow (TF) and Measurement Based Stepped Care (MBSC) Flow Diagrams:**

![image](https://github.com/user-attachments/assets/315183ca-fbf2-45ab-99c8-bb3fa0160e40)
![image](https://github.com/user-attachments/assets/69891e8f-fb0c-4411-b3df-27c8f7584d7a)

## **Team Flow (TF) and Measurement Based Stepped Care (MBSC) Flow Keys:**

![image](https://github.com/user-attachments/assets/e4080867-356f-4efc-a9c4-e9e62134b8e4)
![image](https://github.com/user-attachments/assets/e8e94f3b-7a2b-4386-a25a-2334be2790b5)

## How I tried to locate the variables you are missing

Using these as a reference I will use "TF" when referring to Team Flow and "MBSC" when referring to Measurement Based Stepped Care.

First I tried to Identify each variable you asked for on the diagrams, then I used its identifiers (TF or MBSC) (Associated Letter or Number) to find it in the key, then I checked to make sure it was available in the model File.

## Results

### GMH

#### GMH Step up Rate
 - This variable appears to be referring to "TF-15" or "MBSC-28".
 - Located by those identifiers in the table it's full name is "GMH to SMH Step up Rate".
 - This variable is available in the SP model file.

#### GMH Recommend Step up Rate
 - This variable appears to be referring to "MBSC-1" or "MBSC-16"..
 - Located by those identifiers in the table it's full name is "GMH to SMH Recommend Step up Rate".
 - This variable is available in the SP model file.

#### GMH Recommend New Episode of Care Start Rate
 - This variable appears to be referring to "MBSC-5".
 - Located by those identifiers in the table it's full name is "GMH Recommend New Episode of Care Start Rate".
 - This variable **is NOT** available in the SP model file. (I will need to check with James about this)

#### GMH Recommend Step down Rate
 - This variable appears to be referring to "MBSC-8" or "MBSC-27".
 - Located by those identifiers in the table it's full name is "GMH to PC/PCMHI Recommend Step down Rate".
 - This variable is available in the SP model file. (NOTE: it is named with surrounding quotes "GMH to PC/PCMHI Recommend Step down Rate". I will need to check with James if this is an error in the model.)

#### GMH Recommend Step up Rate (This is a duplicate of the second in this section, but that is how it appeared in your list)
 - This variable appears to be referring to "MBSC-1" or "MBSC-16"..
 - Located by those identifiers in the table it's full name is "GMH to SMH Recommend Step up Rate".
 - This variable is available in the SP model file.

#### GMH Patients Waiting for Step up Rate
 - This variable appears to be referring to the stock "MBSC-L" (which is not a rate).
 - Located by those identifiers in the table it's full name is "GMH Patients Waiting for Step up".
 - This variable is available in the SP model file (NOTE: this is named without surrounding quotes as "GMH to SMH Patients Waiting for Step up").

#### GMH Step Down Rate
 - This variable appears to be referring to "MBSC-33".
 - Located by those identifiers in the table it's full name is "GMH to PC/PCMHI Step down Rate".
 - This variable is available in the SP model file  (NOTE: it is named with surrounding quotes "GMH to PC/PCMHI Step down Rate". I will need to check with James if this is an error in the model).


### SMH

#### SMH High Risk Flag Patient Stepped Care Rate
 - This variable appears to be referring to a rate associated with "TF-A". There is no SMH rate associated with "TF-A"
 - This variable does not exist, and is not needed based on the wireframes.

#### SMH Step up Rate
 - This variable appears to be referring to a rate associated with "TF-G" or "MBSC-N". There are no SMH rates associated with "TF-G" or "MBSC-N".
 - This variable does not exist, and is not needed based on the wireframes.

#### SMH Recommend Step up Rate
 - This variable appears to be referring to a rate associated with "MBSC-A" or "MBSC-H". There are no SMH rates associated with "MBSC-A" or "MBSC-H".
 - This variable does not exist, and is not needed based on the wireframes.

#### SMH Recommend New Episode of Care Start Rate
 - This variable appears to be referring to "MBSC-4".
 - Located by those identifiers in the table it's full name is "SMH Recommend New Episode of Care Start Rate".
 - This variable **is NOT** available in the SP model file. (I will need to check with James about this)

#### SMH Recommend Step down Rate
 - This variable appears to be referring to "MBSC-7", "MBSC-9", "MBSC-25", and "MBSC-26".
 - Located by those identifiers in the table, this is actually two separate variables with full names "SMH to GMH Recommend Step down Rate" and "SMH to PC/PCMHI Recommend Step down Rate".
 - These variables are available in the SP model file, but there are multiple entries (SMH to GMH Recommend Step down Rate, SMH to GMH Recommend Step down Rate2, "SMH to PC/PCMHI Recommend Step down Rate", "SMH to PC/PCMHI Recommend Step down Rate2"), so I need to verify with James which is correct.

#### SMH Patients Waiting for Step up Rate
 - This variable appears to be referring to the stock "MBSC-L" (which is not a rate). There are no SMH variables associated with this stock, since no patients can step up from SMH (it is the highest ranked category).
 - This variable does not exist, and is not needed based on the wireframes.

#### SMH Step Down Rate
 - This variable appears to be referring to "MBSC-31" and "MBSC-32".
 - Located by those identifiers in the table, this is actually two separate variables with full names "SMH to GMH Step down Rate" and "SMH to PC/PCMHI Step down Rate".
 - These variables are available in the SP model file, but there are multiple entries (SMH to GMH Step down Rate, SMH to GMH Step down Rate2, "SMH to PC/PCMHI Step down Rate", "SMH to PC/PCMHI Step down Rate2"), so I need to verify with James which is correct.

### PC/PCMHI

#### PC/PCMHI High Risk Flag Patient Stepped Care Rate
 - This variable appears to be referring to "TF-2".
 - Located by those identifiers in the table it's full name is "PC/PCMHI High Risk Flag Patient Stepped Care Rate".
 - This variable **is NOT** available in the SP model file. (I will need to check with James about this)
 
#### PC/PCMHI Step up Rate
 - This variable appears to be referring to "TF-16", "MBSC-29", and "MBSC-30".
 - Located by those identifiers in the table, this is actually two separate variables with full names "PC/PCMHI to GMH Step up Rate" and "PC/PCMHI to SMH Step up Rate".
 - These variables are available in the SP model file, but there are multiple entries ("PC/PCMHI to SMH Step up Rate", "PC/PCMHI to GMH Step up Rate", "PC/PCMHI to GMH Step up Rate2"), so I need to verify with James which is correct.

#### PC/PCMHI Recommend Step up Rate
 - This variable appears to be referring to "MBSC-2", "MBSC-3", "MBSC-17", and "MBSC-18".
 - Located by those identifiers in the table, this is actually two separate variables with full names "PC/PCMHI to SMH Recommend Step up Rate" and "PC/PCMHI to GMH Recommend Step up Rate".
 - These variables are available in the SP model file, but there are multiple entries ("PC/PCMHI to GMH Recommend Step up Rate", "PC/PCMHI to GMH Recommend Step up Rate2", "PC/PCMHI to SMH Recommend Step up Rate", "PC/PCMHI to SMH Recommend Step up Rate2"), so I need to verify with James which is correct.

#### PC/PCMHI Recommend New Episode of Care Start Rate
 - This variable appears to be referring to "MBSC-6".
 - Located by those identifiers in the table it's full name is "PC/PCMHI Recommend New Episode of Care Start Rate".
 - This variable **is NOT** available in the SP model file. (I will need to check with James about this)

#### PC/PCMHI Recommend Step down Rate
 - This variable appears to be referring to a rate associated with "MBSC-C" and "MBSC-K". There are no PC/PCMHI rates associated with "MBSC-C" and "MBSC-K".
 - This variable does not exist, and is not needed based on the wireframes.

#### PC/PCMHI Patients Waiting for Step up Rate
 - This variable appears to be referring to the stock "MBSC-L" (which is not a rate).
 - Located by those identifiers in the table it's full name is "PC/PCMHI Patients Waiting for Step up".
 - This variable **is NOT** available in the SP model file. (I will need to check with James about this)

#### PC/PCMHI Step Down Rate
 - This variable appears to be referring to a rate associated with "MBSC-O". There are no PC/PCMHI rates associated with "MBSC-O".
 - This variable does not exist, and is not needed based on the wireframes.

**Comment by matthewtomo** [comment]
@tejaspesquare
- Here is an updated list of the SP variables after I checked in with James:
[Corrected.(With_James).Variables-Flow.Meter.Development.for.Team.Flow.SP.txt](https://github.com/user-attachments/files/16821212/Corrected.With_James.Variables-Flow.Meter.Development.for.Team.Flow.SP.txt)

**Comment by matthewtomo** [comment]
**Discussed Dev Standup wk_1 2024_9**
@matthewtomo @tejaspesquare 

- Some of the dials for Team Flow are showing values of zero.
- Current implementation is available in DEV.
- @matthewtomo will look for a data file with more substantial SP numbers to make sure the diagram can be evaluated and validated.

**Comment by matthewtomo** [comment]
**Discussed Dev Standup wk_2 2024_9**
@matthewtomo @tejaspesquare 

- Currently, Dev is waiting on a substantial SP data parameter file so that we can validate the Team Flows implementation.
- [ ] @matthewtomo will attempt to get a data file at the workflows meeting later today, and report back here.

**Comment by matthewtomo** [comment]
**Discussed Friday Dev Standup wk_2 2024_9**
@matthewtomo @tejaspesquare 

- Minor bug with overlapping visuals should be fixed in DEV environment.
- [ ] @matthewtomo still owes @tejaspesquare a substantial SP data file for testing the Team Flow - Flow Diagram.
- Once @matthewtomo gets that data file to @tejaspesquare, the dev team can work on more validation on their end.

**Comment by matthewtomo** [comment]
**Discussed Friday Dev Standup wk_3 2024_09**
@matthewtomo @tejaspesquare 

- [ ] @matthewtomo will set up time with @jamesmrollins to test the Team Flow (SP) Flow diagrams, as the last attempt at generating an SP file failed.
- Implementation for maxDialValue is now complete, and should be implemented for all flow diagrams in TEST environment.

fyi: @jamesmrollins @ljmoody

**Comment by matthewtomo** [comment]
**Discussed Friday Dev Standup wk_4 2024_09**
@matthewtomo @tejaspesquare 

- [ ] @matthewtomo still owes time on testing this
> [ ] @matthewtomo will set up time with @jamesmrollins to test the Team Flow (SP) Flow diagrams, as the last attempt at generating an SP file failed.

**Comment by lzim** [comment]
@matthewtomo 
@jamesmrollins 

_Can we try to avoid holding up **Team Care** just because **Team Flow** is having fails?_
